examples/tv-casting-app/linux/third_party/connectedhomeip

```diff
@@ -0,0 +1 @@
+../../../../
```

Comment:
This appears to be a symlink. Why?

Reply:
Same as most other examples (see https://github.com/project-chip/connectedhomeip/blob/master/examples/lighting-app/linux/third_party/connectedhomeip or https://github.com/project-chip/connectedhomeip/blob/master/examples/all-clusters-app/linux/third_party/connectedhomeip). I think it is for convenience? READMEs carry the following activation instruction: "source third_party/connectedhomeip/scripts/activate.sh"